content/forgero-vanilla/src/main/resources/data/forgero/packs/minecraft-material/type/material/tool_material.json

```diff
@@ -2,12 +2,6 @@
   "parent": "UPGRADE_MATERIAL",
   "name": "TOOL_MATERIAL",
   "children": [
-    {
```

Comment:
Same as above. If we make it a separate type we don't have to break this one up.